packages/database-editor/src/components/ImportWizard/WizardContent.tsx

```diff
@@ -38,24 +38,13 @@ export const WizardContent = ({
         >
           {t('import.back')}
         </Button>
-        {activePage !== pages.length - 2 ? (
-          <Button disabled={!pages[activePage]?.requiredData} variant='primary' size='xl' onClick={() => updateActivePage()}>
-            {activePage === pages.length - 1 ? t('import.close') : t('import.next')}
-            <IvyIcon icon={IvyIcons.Chevron} />
-          </Button>
-        ) : (
-          <Button
-            disabled={!pages[activePage]?.requiredData}
-            variant='primary'
-            size='xl'
-            onClick={() => {
-              creationFunction.mutate();
-              updateActivePage();
-            }}
-          >
-            {t('import.create')}
-          </Button>
-        )}
+        <ProceedButton
+          disabled={!pages[activePage]?.requiredData}
+          create={activePage === pages.length - 2}
+          close={activePage === pages.length - 1}
```

Comment:
better, I think I would also move this logic inside the component, but IMO much better to read 👍